server/utils/dateUtils.test.ts

```diff
@@ -2,6 +2,18 @@ import type { ObjectWithDateParts } from '@approved-premises/ui'
 import { DateFormats, InvalidDateStringError, dateAndTimeInputsAreValidDates, dateIsBlank } from './dateUtils'
 
 describe('DateFormats', () => {
+  describe('formatApiDate', () => {
+    it('converts a date object to a ISO8601 date string', () => {
+      expect(DateFormats.formatApiDate(new Date(2022, 4, 11))).toEqual('2022-05-11')
```

Comment:
We have changed the DateFormats functions to all be `xToY` as opposed to having some as `formatX`.